tests/foreman/api/test_syncplan.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=invalid-name
```

Comment:
i am in favor of shortening names to common abbreviations and context-based word removal. So ```test_create_sync_plan_with_random_name``` could be changed to ```create_with_rand_name```. This also strips test from the name which could go either way.